core/common/src/main/java/alluxio/exception/runtime/BlockDoesNotExistRuntimeException.java

```diff
@@ -9,29 +9,22 @@
  * See the NOTICE file distributed with this work for information regarding copyright ownership.
  */
 
-package alluxio.exception;
-
-import alluxio.grpc.ErrorType;
-
-import io.grpc.Status;
+package alluxio.exception.runtime;
 
 import java.text.MessageFormat;
 
 /**
  * The exception thrown when a block does not exist in Alluxio.
  */
-public class BlockDoesNotExistRuntimeException extends AlluxioRuntimeException {
-  private static final long serialVersionUID = -1313208091357063634L;
-  private static final Status STATUS = Status.NOT_FOUND;
-  private static final ErrorType ERROR_TYPE = ErrorType.User;
+public class BlockDoesNotExistRuntimeException extends NotFoundRuntimeException {
 
   /**
    * Constructs a new exception with the specified detail message.
    *
    * @param message the detail message
    */
   public BlockDoesNotExistRuntimeException(String message) {
```

Comment:
This API doesn't seem to be necessary. If a block does not exist exception takes an arbitrary message, caller might as well use not found directly.